frontend/src/components/personal/ProfilePreferencesPage.styles.tsx

```diff
@@ -38,5 +39,37 @@ export const useStyles = makeStyles((theme) => ({
     },
     textfields: {
       margin: 15,
-    }
-  }))
\ No newline at end of file
+    },
+    
+  }))
+
+  export const useStyleModal = makeStyles((theme: Theme) =>
+  createStyles({
+    paper: {
+      position: 'absolute',
+      width: 400,
+      backgroundColor: theme.palette.background.paper,
+      border: '1px dotted #000',
+      boxShadow: theme.shadows[5],
+      padding: theme.spacing(2, 4, 3),
+      fontFamily: 'Roboto',
+      lineHeight: '1.5rem'
+    },
+  }),
+)
+
+export function rand() {
+  return Math.round(Math.random() * 20) - 10
+}
+
+export function getModalStyle() {
+  const top = 20 + rand()
```

Comment:
Why do we use random for positioning?

Reply:
This way position on screen looks better. Its taken from Material UI code example for Modal window.

Reply:
It does not sounds clear to me: how random factor improves look&feel on screen? Why we can't just set fixed positioning?